webdriver/tests/element_click/support/close_window.html

```diff
@@ -0,0 +1,9 @@
+<html>
+    <body onload="checkload()">
+        <script>
+        function checkload() {
+            window.close()
+        }
+        </script>
+    </body>
+</html>
```

Comment:
Mind adding a final empty line so the linter will be happy?

Reply:
fixed